snooty/postprocess.py

```diff
@@ -509,8 +517,13 @@ def enter_node(self, fileid_stack: FileIdStack, node: n.Node) -> None:
             self.current_depth += 1
             return
 
-        if isinstance(node, n.Directive) and node.name == "method-option":
-            self.scanned_pattern.append(node.options["id"])
+        # add tab selectors here as well
+
+        if isinstance(node, n.Directive):
+            if node.name == "method-option":
+                self.scanned_pattern.append((node.name, node.options["id"]))
+            elif node.name == "tab":
```

Comment:
just noting that collapsibles here are not required since we can expand and scroll to headings within collapsibles